floss/main.py

```diff
@@ -420,7 +423,7 @@ def filter_unique_decoded(decoded_strings):
     unique_values = set()
     originals = []
     for decoded in decoded_strings:
-        hashable = (decoded.va, decoded.s, decoded.decoded_at_va, decoded.fva)
+        hashable = (decoded.s, decoded.decoded_at_va, decoded.fva)
```

Comment:
i presume that you made this fix because in practice our routine was *too* conservative in detecting duplicates?

Reply:
Removing the string va makes sense, although I'm not quite sure this filters out many duplicates.

Reply:
The reason for not having decode.va part of the hash is that even so two equal strings are decoded by the same function and at the same decoded.decoded_at_va they will have diffrent decoded.va addresses. I actually have no idea why. This only happens if the decoding is done in a loop which is why it is exposed when using the --max-address-revisits. Do you have any insight into why vivisect does this ?